.github/workflows/build.yaml

```diff
@@ -42,18 +50,17 @@ jobs:
       - name: Build Binary
         run: cargo build --locked --release --target ${{ matrix.target }}
 
-      - name: Upload artifact
-        uses: actions/upload-artifact@v4
+      - name: Rename Binary for target
+        if: matrix.name != 'win-amd64'
+        run: mv target/${{ matrix.target }}/release/mesh-thumbnail* target/${{ matrix.target }}/release/mesh-thumbnail-${{ matrix.target }}
+      # Pesky .exe suffix. There's probably a better way to do this.
+      - name: Rename Binary for Windows
         if: matrix.name == 'win-amd64'
-        with:
-          name: Windows-binary
-          path: |
-            target/${{ matrix.target }}/release/mesh-thumbnail.exe
+        run: mv target/${{ matrix.target }}/release/mesh-thumbnail.exe target/${{ matrix.target }}/release/mesh-thumbnail-${{ matrix.target }}.exe
```

Comment:
Why are you renaming things?

Reply:
because it'll be difficult to tell which OS/arch each thing is for in the Assets of the release if they don't get renamed.
3x `mesh-thumbnail` would be confusing here, if it's even possible.
![Screenshot 2025-04-08 at 22 25 28](https://github.com/user-attachments/assets/a6f5692a-b819-4b0a-b3cd-f5fb06b7c144)

Reply:
I see where this'll need to get change in mesh-organizer, I'll put up a PR for that too.